tests/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ target_link_libraries(
   simple-dll
 )
 
+set_source_files_properties(large-alignment-test.c PROPERTIES COMPILE_OPTIONS "-fno-lto")
```

Comment:
@eukarpov Do you know the details of the root cause of this issue? Is it already reported in the backlog?